website/source/docs/agent/telemetry.html.md

```diff
@@ -649,12 +724,72 @@ These metrics give insight into the health of the cluster as a whole.
     <th>Unit</th>
     <th>Type</th>
   </tr>
+  <tr>
+  <td>`consul.memberlist.degraded.probe`</td>
+  <td>This metric counts the number of times the agent has performed failure detection on an other agent at a slower probe rate. The agent uses its own health metric as an indicator to perform this action. (If its health score is low, means that the node is healthy, and vice versa.)</td>
```

Comment:
I thought Freddy's suggestion for this was well worded and included a useful link maybe we could just replace it with his:

> A node's perception of its own health based on how well it is meeting the soft real-time requirements of the protocol. This metric ranges from 0 to 8, where 0 indicates "totally healthy". This health score is used to scale the time between outgoing probes, and higher scores translate into longer probing intervals. For more details see section IV of the Lifeguard paper: https://arxiv.org/pdf/1707.00788.pdf

Reply:
Let me replace this.